webpack/webpack.config.base.js

```diff
@@ -16,28 +16,24 @@ const ProgressBarPlugin = require('progress-bar-webpack-plugin');
 
 const SRC_ROOT_PATH = path.join(__dirname, '../', 'src');
 const DIST_ROOT_PATH = path.join(__dirname, '../', 'dist');
+
 const NODE_ENV = process.env.NODE_ENV || 'development';
-const IS_DEV = NODE_ENV === 'development';
-const IS_PROD = !IS_DEV;
+const IS_DEV = process.env.NODE_ENV === 'development';
```

Comment:
```suggestion
const IS_DEV = NODE_ENV === 'development';
```
don't we wanna use the default here?